_episodes/07-docker-image-examples.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Examples of Using Container Images in Practice"
+teaching: 20
+exercises: 0
+questions:
+- "How can I use Docker for my own work?"
+objectives:
+- "Use existing container images and Docker in a research project."
+keypoints:
+- "There are many ways you might use Docker and existing container images in your research project."
+---
+
+Now that we have learned the basics of working with docker images and containers,
+let's apply what we learned to an example workflow.
+
+You may choose one or more of the following examples to practice using containers.
+
+## Jekyll Website Example
+
+In this [Jekyll Website example](../_extras/e02-jekyll-lesson-example), you can practice
+rendering this lesson website on your computer using Jekyll in a Docker container.
+Rendering the website in a container avoids a complicated software installation; instead of installing Jekyll and all the other tools needed to create the final website, all the work can be done in the container.
+
```

Comment:
Perhaps could add the benefit of clean-up? (I don't feel strongly about this so feel free to jettison the suggestion!)
```suggestion
Also, when you no longer need to render the website, you can easily and cleanly remove the software from your computer.
```

Reply:
```suggestion
Additionally, when you no longer need to render the website, you can easily and cleanly remove the software from your computer.
```